src/main/java/uk/gov/hmcts/reform/sscs/helper/mapping/HearingsDetailsMapping.java

```diff
@@ -95,6 +99,28 @@ public static boolean shouldBeHearingsInWelshFlag() {
 
     public static List<HearingLocation> getHearingLocations(SscsCaseData caseData,
                                                             ReferenceDataServiceHolder referenceDataServiceHolder) {
+        List<HearingLocation> hearingLocations = getAllHearingLocations(caseData, referenceDataServiceHolder);
+
+        String nextHearingVenueName = caseData.getAdjournCaseNextHearingVenue();
+
+        if (!referenceDataServiceHolder.isAdjournmentFlagEnabled() && isNotEmpty(nextHearingVenueName)
+            && !HearingsChannelMapping.isPaperCase(caseData)) {
+            try {
+                String venueID = getVenueID(caseData, nextHearingVenueName);
+
```

Comment:
Del asked me to log the id of the case and what was calculated and why. Might be relevant here.

Reply:
Done